services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: str, movie_id: int, cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        year, month, day = session_date.split("-")
+        return MovieSession.objects.filter(
+            show_time__year=year, show_time__month=month, show_time__day=day
```

Comment:
You can use suffix __date to get access to date of the DateTimeField (read hints in task :) )

Reply:
@OlegatorLE thank youuu!